Mathlib/Algebra/BigOperators/NatAntidiagonal.lean

```diff
@@ -22,6 +22,27 @@ namespace Finset
 
 namespace Nat
 
+section Binders
+open Mathlib.FlexibleBinders Lean Macro
+
+/-- For the `finset` domain, `a + b = n` for sums over the antidiagonal.
+Example: `∑ i + j = 10, i * j`.
+
+To avoid generating a `match` expression, it generates a name based on the two
+variables and substitutes projections.
+The example expands as `∑ ij ∈ Nat.antidiagonal 10, ij.1 * ij.2`. -/
```

Comment:
```suggestion
The example expands as `∑ ij ∈ Finset.antidiagonal 10, ij.1 * ij.2`. -/
```
`Nat.antidiagonal` no longer exists (which also suggests this now belongs in a different file)